spec/lib/searching/result_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe Searching::Result do
+  let(:sample_data) do
```

Comment:
This top level `let` is never used, can we delete it

Reply:
done